rules/EarlyReturn/Rector/If_/ChangeNestedIfsToEarlyReturnRector.php

```diff
@@ -73,71 +76,90 @@ public function run()
      */
     public function getNodeTypes(): array
     {
-        return [If_::class];
+        return NodeTypeGroup::STMTS_AWARE;
     }
 
     /**
-     * @param If_ $node
+     * @param Function_|ClassMethod $node
```

Comment:
`STMTS_AWARE` is not only `Function_` and `ClassMethod`, but 

```
ClassMethod::class, Function_::class, If_::class, Else_::class, ElseIf_::class, Do_::class, Foreach_::class, TryCatch::class, While_::class
```

Reply:
Yeah, I think we'll need to introduce some common interface that all these node should implements, so we can avoid naming extreemely long union here.